peps/conf.py

```diff
@@ -71,6 +71,7 @@
     "typing": ("https://typing.readthedocs.io/en/latest/", None),
     "trio": ("https://trio.readthedocs.io/en/latest/", None),
     "devguide": ("https://devguide.python.org/", None),
+    "mypy": ("https://mypy.readthedocs.io/en/latest/", None),
```

Comment:
```suggestion
    "devguide": ("https://devguide.python.org/", None),
    "mypy": ("https://mypy.readthedocs.io/en/latest/", None),
    "typing": ("https://typing.readthedocs.io/en/latest/", None),
    "trio": ("https://trio.readthedocs.io/en/latest/", None),
```

Reply:
I reordered everything, let me know if this is good